modules/ROOT/pages/major-changes.adoc

```diff
@@ -0,0 +1,144 @@
+= Major Changes in Fedora CoreOS
+:toc:
+
+This is a list of major changes that were introduced in Fedora CoreOS with the
+notes associated with them.
+Those changes are also announced on the https://lists.fedoraproject.org/archives/list/coreos-status@lists.fedoraproject.org/[coreos-status mailing list].
+Note that this is not an exhaustive list of changes in Fedora CoreOS and only includes major changes that may require manual actions.
+This list is in reverse chronological order to keep recent changes at the top.
+
+////
```

Comment:
Minor: looks like the GitHub syntax renderer is confused by this and everything after the template isn't highlighted correctly. Not sure if there's a way to fix that. I guess we could put the template at the bottom of the doc instead?